pkg/ingester/owned_series.go

```diff
@@ -65,27 +64,10 @@ func newOwnedSeriesService(interval time.Duration, instanceID string, ingesterRi
 		}),
 	}
 
-	oss.Service = services.NewTimerService(interval, oss.starting, oss.onPeriodicCheck, nil)
+	oss.Service = services.NewTimerService(interval, nil, oss.onPeriodicCheck, nil)
 	return oss
 }
 
-// This is Starting function for ownedSeries service. Service is only started after all TSDBs are opened.
-// Pushes are not allowed yet when this function runs.
-func (oss *ownedSeriesService) starting(ctx context.Context) error {
-	err := ring.WaitInstanceState(ctx, oss.ingestersRing, oss.instanceID, ring.ACTIVE)
```

Comment:
I thought the change would be here... instead of waiting for ACTIVE state, we would just wait for entry in the ring with any state.

Reply:
As mentioned in the above comment, we want to avoid increasing the delay between the lifecycler starting and the ingester service moving to `Running`.